package.json

```diff
@@ -20,7 +20,7 @@
     "test": "jest",
     "test:ci": "yarn lint && yarn type-check && yarn test",
     "build": "tsc -p tsconfig.dist.json --declaration",
-    "prepush": "yarn lint && yarn type-check && jest --changedSince master",
+    "prepush": "yarn lint && yarn type-check && jest --changedSince main",
     "prepack": "yarn build"
   },
   "peerDependencies": {
```

Comment:
Please update this project to the last sdk-* versions.

Reply:
@VDubber I think this project is pretty far behind. I think we will need a separate task for upgrading the integration. In the meantime, I think we should go ahead and release this.